rpms/awips2.edex/Installer.edex/component.spec

```diff
@@ -62,24 +62,10 @@ if [ $? -ne 0 ]; then
    exit 1
 fi
 
-mkdir -p %{_build_root}/awips2/edex/data/ndm
-if [ $? -ne 0 ]; then
-   exit 1
-fi
-
-mkdir -p %{_build_root}/awips2/dev
-if [ $? -ne 0 ]; then
-   exit 1
-fi
-
-mkdir -p %{_build_root}/awips2/dev/logs
-if [ $? -ne 0 ]; then
-   exit 1
-fi
-
 /bin/cp -r %{_baseline_workspace}/rpms/awips2.edex/Installer.edex/programs/qpidNotify.py ${RPM_BUILD_ROOT}/awips2/edex/bin/
 
 /bin/cp -r %{_baseline_workspace}/rpms/awips2.edex/Installer.edex/programs/updateNDM.pl ${RPM_BUILD_ROOT}/awips2/dev/
+/bin/cp -r %{_baseline_workspace}/rpms/awips2.edex/Installer.edex/programs/logs ${RPM_BUILD_ROOT}/awips2/dev/
```

Comment:
this looks slightly different than before?  So now we're creating a different logs directory? and using `$RPM_BUILD_ROOT` instead of `$_build_root` is that okay?  Is there a reason for the difference?

Reply:
Like I said above, the mkdir -p lines weren't doing anything that I could tell. The directories were getting created by the copy. The only line that didn't have a copy before was the logs and that directory didn't exist/get created on the RPM install. Now it should with the copy command